examples/generative/corrdiff/training/training_loop.py

```diff
@@ -181,9 +186,22 @@ def training_loop(
         logger0.success("Loaded the pre-trained regression network")
     else:
         interface_kwargs = {}
-    loss_fn = construct_class_by_name(**loss_kwargs, **interface_kwargs)
+
+    if (loss_kwargs.class_name == "modulus.metrics.diffusion.ResLoss"):
+        loss_fn = construct_class_by_name(**loss_kwargs, **interface_kwargs,
+        img_shape_x = img_shape_x,
```

Comment:
I suggest we include these kwargs as part of the interface_kwargs. Then `construct_class_by_name(**loss_kwargs, **interface_kwargs` will work for any loss.